qcenginerecords/molpro/ccsd_prt_pr_water_gradient/output.json

```diff
@@ -49,7 +49,10 @@
     "scf_total_energy": -74.9646625166939,
     "ccsd_prt_pr_correlation_energy": -0.0550552030560323,
     "ccsd_prt_pr_total_energy": -75.0197177197499,
-    "ccsd_prt_pr_dipole_moment": [0.0, 0.0, 0.606516899290546]
+    "ccsd_prt_pr_dipole_moment": [0.0, 0.0, 0.606516899290546],
+    "calcinfo_nalpha": 5,
+    "calcinfo_nbeta": 5,
+    "calcinfo_nbasis": 7
```

Comment:
Cool, great to add this to the parsing!

Reply:
Yes, unfortunately Molpro doesn't write out the nalpha and nbeta explicitly but I think I've figured out a workaround that still only needs the xml and the assumption that `nalpha > nbeta` which I believe is always true in Molpro.

Reply:
That is the assumption in most programs that I have worked with. It doesn't matter and might as well pick one.